### PR TITLE
Initial theming

### PR DESCRIPTION
**Pull request by betovarg**
Initial theming with styles from Kendall and my own. Inner pages have functional and responsive styles.
Created file and folder structure for the theme.

### CODE REVIEW

File: sites/all/modules/features/dccr_ftr_sponsors/dccr_ftr_sponsors.views_default.inc [hunk 1]
```diff
@@ -74,7 +74,7 @@ function dccr_ftr_sponsors_views_default_views() {
 
   /* Display: Page */
   $handler = $view->new_display('page', 'Page', 'page');
-  $handler->display->display_options['path'] = 'sponsors';
+  $handler->display->display_options['path'] = 'sponsors-view';
```
Comment (kporras07) [inline sites/all/modules/features/dccr_ftr_sponsors/dccr_ftr_sponsors.views_default.inc:77]: nice :)

**Comment by kporras07** [comment]
@betovarg :shipit:

**Comment by betovarg** [comment]
lol, had to create a new page for that, but did not wanted to delete that right now.

**Comment by betovarg** [comment]
@kporras07 The page for sponsors was created with blocks and context. For it to look like the rest of the inner pages, I would have had to write a lot of new layout code. There was a panel for sponsors, so I added that /sponsors url to the panel and changed the url for the view page.

**Pull request merged**